src/util/np_utils.py

```diff
@@ -26,3 +26,78 @@ def angle_to_rotate(v1, v2):
     if sign == 0.0:
         sign = 1
     return smallest_angle * sign
+
+
+def intersect_2d(start1: np.array, end1: np.array, start2: np.array, end2: np.array) -> bool:
```

Comment:
is this actually used anywhere?

Reply:
Nope, it was written before I started using shapely. I figured it might be useful to someone at some point so I kept this (and orientation) in, but I can delete them if you'd like.

Reply:
nah its ok should be fine